doc/stdlib/math.rst

```diff
@@ -1696,7 +1704,7 @@ Mathematical Functions
 
    .. Docstring generated from Julia source
 
-   Compute the digamma function of ``x`` (the logarithmic derivative of ``gamma(x)``\ )
+   Compute the digamma function of ``x`` (the logarithmic derivative of :func:`gamma`\ ).
```

Comment:
It's the expected behaviour at the moment. Links converted from md to rst just get replaced with the URL if they match the ``:thing:`...` `` syntax, so those markdown links should be written as ``[`gamma(x)`](:func:`gamma(x)`)`` if you'd like the argument to be kept.